themes/hugo-rladies/layouts/events/single.html

```diff
@@ -1,335 +1,164 @@
-        "useCreationPopup": false,
-        "month": {
-          "startDayOfWeek": 1, // monday
-        },
-        "week": {
-          "startDayOfWeek": 1, // monday
+
+  <div id="calendar" style="width: 100%; height: 600px"></div>
+  <small class="text-muted"></small>
+</article>
+{{ end }} {{ define "footer" }}
+<script type="text/javascript">
+  // var window_width = document.documentElement.clientWidth;
+  document.addEventListener("DOMContentLoaded", function () {
+    var timeZoneSelectorEl = document.getElementById("time-zone-selector");
+    timeZone = document.getElementById("time-zone-selector").value;
+    var loadingEl = document.getElementById("loading");
+    var calendarEl = document.getElementById("calendar");
+    evnt = {
+      url: "https://raw.githubusercontent.com/rladies/website/main/data/events.json", //your url,
+      type: "GET",
+      dataType: "json",
+      success: function (data) {
+        //based on the dropdown changetimezone of each event
+        //console.log(data);
+
+        let updatedTime = [];
+        $.each(data, function (k, v) {
+          v.start = moment.tz(v.start, timeZone).format();
+          v.end = moment.tz(v.end, timeZone).format();
+          updatedTime[k] = v;
+        });
+        //console.log(Intl.DateTimeFormat().resolvedOptions().timeZone);
+        return updatedTime;
+      },
+    };
+    var calendar = new FullCalendar.Calendar(calendarEl, {
+      //schedulerLicenseKey: 'CC-Attribution-NonCommercial-NoDerivatives',
+      views: {
+        timeGridUseRWeek: {
+          type: "timeGridWeek",
+          //duration: { days: 5 },
+          visibleRange: {
+            start: "2019-12-17",
+            end: "2021-09-30",
+          },
+          buttonText: "week",
         },
-        "isReadOnly": true,
-        "usageStatistics": false,
-        "useNav": true,
-        "theme": rladies_cal,
-        "calendars":  [
-          {{ range .Site.Data.chapters }}
-            {
-              id: {{ .id }},
-              name: {{ .name | default "rladies" }},
-              color: {{ .color | default "#fff" }},
-              bgColor: {{ .bgColor | default "#88398a" }},
-              borderColor: {{ .color | default "#88398a" }},
-            },
-          {{ end }}
-        ] 
-      });
-  
-      function cal_change_ww(size){
-        var viewName = "";
-        var options = cal.getOptions();
-        switch (true) {
-            case (size < 650):
-                viewName = 'day';
-                break;
-            case (size < 900):
-                viewName = 'week';
-                break;
-            case (size < 1200):
-                options.month.visibleWeeksCount = 2;
-                viewName = 'month';
-                break;
+      },
+      headerToolbar: {
+        left: "prev,next",
+        center: "title",
+        right: "dayGridMonth,timeGridUseRWeek,listWeek,timeGridDay",
+      },
+      initialDate: "2021-01-04",
```

Comment:
```suggestion
      initialDate: moment().format("YYYY-MM-DD")
```
Start view on current date